agilent_infiniium_scope.py

```diff
@@ -207,43 +207,48 @@ def averagemode(self, c, mode = None):
                 elif isinstance(mode, bool):
                     mode = int(mode)
                 elif isinstance(mode, int):
-                    pass       
+                    pass
             yield dev.write('ACQ:AVER '+str(mode))
             resp = yield dev.query('ACQ:AVER?')
-        returnValue(resp) 
-        
+        returnValue(resp)
+
     @setting(119, navg = 'i', returns = ['i'])
     def numavg(self, c, navg = None):
         """Get or set number of averages
         """
         dev = self.selectedDevice(c)
         if navg is None:
             resp = yield dev.query('ACQ:COUN?')
-        else:    
+        else:
             yield dev.write('ACQ:COUN %d' %navg)
             resp = yield dev.query('ACQ:COUN?')
         navg_out = int(resp)
-        returnValue(navg_out)         
+        returnValue(navg_out)
 
     @setting(131, channel = 'i', level = 'v', returns = 'v{level}')
-    def trigger_at(self, c, channel, level = None):
-        """Get or set the trigger source and the trigger voltage for edge mode triggering.
-        Channel must be one of 0 (AUX), 1, 2, 3, 4.
+    def trigger_at(self, c, channel, level=None):
+        """Get or set the trigger source and the trigger voltage for edge mode
+         triggering.
+        Channel must be one of 0 (AUX), 1, 2, 3, 4, 5(LINE).
```

Comment:
Nit: docstring format says one line summary, then blank line, then rest. You can shorten the first line a bit, e.g. 
```python
"""Get or set the trigger source and voltage for edge triggering.

Channel must be one of 0 (AUX), 1, 2, 3, 4, or 5 (LINE).
"""